config/nut/nut.inc

```diff
@@ -333,7 +326,7 @@ EOD;
 EOD;
 
 		$start = <<<EOD
-if [ `pgrep {$driver} | wc -l` != 0 ]; then
+if [ `/bin/pgrep {$driver} | /usr/bin/wc -l` != 0 ]; then
```

Comment:
My suggestion didn't use wc, it only uses pgrep return code, that will be 0 (success) if process is running. But nvm, I'll take care of these cases later, maybe change all of them in one shot. I like to change these cases because current code runs multiple commands just to check if process is running. It runs [ (test), then start a new sh session due to backticks, then run pgrep and wc.